breakout_3_comments.md

```diff
@@ -0,0 +1,16 @@
+# Reviewer Comments
```

Comment:
storyline for breakout 3 is:
- after making revisions, paper got rejected. 😢 
- decide to submit to another journal (Journal of Biomechanics - really the next best option besides Nature 😉 ).
- undo the bad reviewer's comments and make some final touches before re-submission.